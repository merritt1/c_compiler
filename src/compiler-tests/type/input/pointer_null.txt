int main(void) {
signed int *p;
(((signed int *)0) == p);
}
