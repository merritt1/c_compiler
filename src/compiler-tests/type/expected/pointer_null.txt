=============== PARSE TREE ===============
signed int (main(void)) {
signed int (*(p));
((signed int(*)) 0==p);
}
================= SYMBOLS ================
Symbol Table ID: 0
  main -> function(signed int, [void])

Symbol Table ID: 1
  p -> pointer(signed int)

